apps/site/lib/site/body_tag.ex

```diff
@@ -33,7 +33,7 @@ defmodule Site.BodyTag do
   end
 
   defp enable_turbolinks?(conn) do
-    is_nil(conn.assigns[:disable_turbolinks]) or Turbolinks.enabled?(conn)
+    is_nil(conn.assigns[:disable_turbolinks])
```

Comment:
The perhaps-poorly-named `Turbolinks.enabled?` checks whether the page being rendered was requested via Turbolinks. So, it would always be false for the first page a user landed on, then would permanently (until the next full page load) flip to true the next time the user followed a link, as long as their current page hadn't explicitly disabled Turbolinks. Whatever the originally intended behavior was, this likely wasn't it.